tests/test_stac.py

```diff
@@ -135,6 +141,11 @@ def test_create_glo90_item(self):
         self.assertEqual(data.media_type, MediaType.COG)
         self.assertEqual(data.roles, ["data"])
 
+        self.assertCountEqual(item.stac_extensions, [
+            pystac.extensions.projection.SCHEMA_URI,
+            pystac.extensions.raster.SCHEMA_URI,
+        ])
+
```

Comment:
```suggestion
        ProjectionExtension.validate_has_extension(item)
        RasterExtension.validate_has_extension(item)
```

Reply:
1) validate_has_extension has a mandatory parameter add_if_missing.
2) `validate` is a bad name for a method that has can modify the object
3) has_extension is more what should be done here.